### PR TITLE
Enhance POC notebook and docs

### PR DESCRIPTION
**Pull request by SYangster**
### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: examples/tutorials/setup_poc.ipynb [hunk 1]
```diff
@@ -104,283 +101,278 @@
...
-    "* **Setup NVFLARE in docker mode**\n",
-    "\n",
-    "Normally, when we using ```nvflare poc start```, the clients and server are started as process in the local host. \n",
-    "What if you would like to experiment runing client and server in dockers ? You can do it now, \n",
+    "The -he option enables homomorphic encryption.\n",
     "\n",
     "```\n",
-    "nvflare poc prepare -n 2 -d  <docker_image>\n",
-    "\n",
+    "nvflare poc prepare -he\n",
```
Comment (YuanTingHsieh) [inline examples/tutorials/setup_poc.ipynb:284]: Changes in this PR LGTM,

I just have a general question:
As POC grows complicated, it is trying to get all the functionalities that our provision has, why don't we just ask people to do provision - deployment after simulator?

**Comment by YuanTingHsieh** [comment]
@SYangster do we want same changes in 2.5?

File: examples/tutorials/setup_poc.ipynb (hunk 1 above)
Comment (SYangster) [reply examples/tutorials/setup_poc.ipynb:284]: Good question. The main thing is POC mode allows you to test all the features of a full FLARE deployment on a single machine (using the same project.yml), but with much less overhead than a real deployment.

**Review by YuanTingHsieh — APPROVED** [review]
LGTM

**Pull request merged**